ovos_utils/log.py

```diff
@@ -276,3 +290,73 @@ def log_wrapper(*args, **kwargs):
         return log_wrapper
 
     return wrapped
+
+
+def get_log_path(service: str, directories: Optional[List[str]] = None) \
+    -> Optional[str]:
+    """
+    Get the path to the log directory for a given service.
+    Default behaviour is to check the configured paths for the service.
+    If a list of directories is provided, check that list for the service log
+
+    Args:
+        service: service name
+        directories: (optional) list of directories to check for service
+
+    Returns:
+        path to log directory for service
+    """
+    from ovos_utils.xdg_utils import xdg_state_home
+    try:
+        from ovos_config import Configuration
+        from ovos_config.meta import get_xdg_base
+    except ImportError:
+        xdg_env = os.environ.get("OVOS_CONFIG_BASE_FOLDER")
+        return os.path.join(xdg_state_home(), xdg_env or "mycroft")
```

Comment:
should this return?

`xdg_env = get_xdg_base() or os.environ...` and resume the rest of the code

Reply:
as a TODO for later

seems that state home util is missing here https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/locations.py

but that should be updated and used directly instead of importing get_xdg_base here and manually creating the path